docker-entrypoint.sh

```diff
@@ -11,11 +11,12 @@ cd /chia-blockchain || exit 1
 # shellcheck disable=SC1091
 . ./activate
 
+# shellcheck disable=SC2086
+chia ${chia_args} init --fix-ssl-permissions
+
 if [[ -n ${ca} ]]; then
-  chia_ca_arg="-c ${ca}"
+  chia init -c "${ca}"
```

Comment:
This `chia` command may also need to have `${chia_args}`
```suggestion
  chia ${chia_args} init -c "${ca}"
```

Reply:
Yeah so.. I can't remember why we added that, but thats on the _other_ init command.. I couldn't think of why we'd need it just for the CA step?